lib/fortnox/api/models/invoice.rb

```diff
@@ -1,142 +1,43 @@
-require "fortnox/api/models/base"
+require "fortnox/api/models/document_base"
 require "fortnox/api/models/edi_information"
-require "fortnox/api/models/email_information"
-require "fortnox/api/models/row"
+require "fortnox/api/models/invoice_row"
```

Comment:
Well, if we are not going to have any other row models we can leave it as just `Row`. About the module argument it depends. If this is specialisations it's fine to use inheritance, but if it's just that they share some properties (which I think is the case here) we should change it into a mixin instead.